x-pack/plugins/siem/server/lib/timeline/routes/import_timelines_route.ts

```diff
@@ -119,35 +119,35 @@ export const importTimelinesRoute = (
                       return null;
                     }
                     const {
-                      savedObjectId,
+                      savedObjectId = null,
                       pinnedEventIds,
                       globalNotes,
                       eventNotes,
                       templateTimelineId,
                       templateTimelineVersion,
                       timelineType,
+                      version = null,
                     } = parsedTimeline;
                     const parsedTimelineObject = omit(
                       timelineSavedObjectOmittedFields,
```

Comment:
Omitting the fields that timeline savedObject doesn't take to avoid failure while creating / updating savedObject.

Reply:
We'll manage globalNotes for template timeline down below on Line 159-161